lib/http/HttpService.ts

```diff
@@ -5,10 +5,12 @@ class HttpService {
   constructor(private service: Service) {}
 
   public resolveHashRaiden = async (resolveRequest: RaidenResolveRequest): Promise<RaidenResolveResponse> => {
-    // TODO: add reveal_timeout, settle time out,  token, etc
+    // TODO: add settle time out, etc
     const secret = await this.service.resolveHash({
       rHash: resolveRequest.secrethash.slice(2),
       amount: resolveRequest.amount,
+      tokenAddress: resolveRequest.token,
+      cltvDelta: resolveRequest.reveal_timeout,
```

Comment:
Value here is not a delta but a block number.

We will need also the lock time and maybe also the height if we know it.

Reply:
Do you mean that the `reveal_timeout` in the resolveRequest is a block number? I'm going off of the raiden REST API docs which says 

> 'reveal_timeout': The maximum number of blocks allowed between the setting of a hashlock and the revealing of the related secret.

Looking at the Raiden code it looks like `expiration` gets set to `BlockExpiration` which sounds more like an actual block number? I'm still not really sure so I will test and debug this code to see what the actual values are.

Reply:
It looks like `reveal_timeout` is the lower bound of lock expiration, and `expiration` is the actual lock expiration for the incoming payment.